badgerdb/badgerdb.go

```diff
@@ -105,8 +138,15 @@ type Options struct {
 // DefaultOptions is an Options object with default values.
 // Dir: "BadgerDB", Codec: encoding.JSON
```

Comment:
I think in all existing store implementations I added the default values here in the comments as well. I'm not sure if that was a good idea, so instead of adding the new values you added to the `DefaultOptions` maybe you can just remove this line for now. If I think I'd like to have it back then I will add it with _all_ values later.